src/components/node-manager.js

```diff
@@ -108,8 +108,11 @@ class NodeManager {
      */
     layoutPropsForId(id) {
         if (!this._cachedBoundingBoxesById[id]) {
-            this._cachedBoundingBoxesById[id] =
-                this._nodesById[id].getBoundingClientRect();
+            const node = this._nodesById[id];
```

Comment:
Is the `id` the DOM id of the node?  If so, maybe we should try querying for it and only if we can't find then return a new `DOMRect()`.

Reply:
This is really difficult to replicate.  Requires me to run in a phone emulator, and a bunch of steps... so I'd rather not try to get too tricky here.